style.css

```diff
@@ -284,6 +284,81 @@ header a {
   transform: scale(1.1);
 }
 
+.contact-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+  width: 100%;
+  background-image: url(/images/contact-form-m-bg.svg);
+  background-size: contain;
+  background-position: top right;
+  background-repeat: no-repeat;
+  padding: 50px 24px;
+}
+
+.contact-section h2 {
+  color: #fff;
+  width: 90%;
+  text-align: center;
+  padding-top: 25px;
+  font-weight: 700px;
+  font-size: 40px;
+}
+
+.contact-section p {
+  color: #ebebff;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  margin-top: -10px;
+  padding: 30px;
+}
+
+.contact-section form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.contact-input {
+  padding: 15px;
+  width: 375px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  color: #172b4d;
+  outline: none;
+}
```

Comment:
Your style is not exactly similar to the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=8P269HoVLs4fzLYn-0). Your width is `375px` where in the template it `is 327px` with the center aligned. Please consider reviewing your styles to make them exact replica of the template so it looks more better and perfect.